docs/utilities/large_messages.md

```diff
@@ -274,23 +332,62 @@ This utility works perfectly together with the batch module (`powertools-batch`)
     }
     ```
 
+!!! tip "Use together with idempotency module"
+    This utility also works together with the idempotency module (`powertools-idempotency`). 
+    You can add both the `@LargeMessage` and `@Idempotent` annotations to the same method. 
+    The `@Idempotent` takes precedence over the `@LargeMessage` annotation. 
+    It means Idempotency module will use the initial raw message (containing the S3 pointer) and not the large message.
+    Using the large message would end up with potential issues when inserting the data in DynamoDB, where items
```

Comment:
Do we need this sentence at all, now that you have organised the precedence?

Reply:
I already say 

> The `@Idempotent` takes precedence over the `@LargeMessage` annotation.

what should we add ?

Reply:
maybe something about our discussion and the deleted file in S3...

Reply:
I think the comment about the precedent is all we need:
> The @Idempotent takes precedence over the @LargeMessage annotation.

This part, explaining (as I understand it) the reason the precedence can't work the other way around, feels unnecessary - with the precedence, you've made sure it doesn't work this way at all, right?

Reply:
> maybe something about our discussion and the deleted file in S3...

Is this a user-facing detail?

Reply:
Not sure to get your comment. But whatever the order you choose for annotations on the method, Idempotency will always be the first executed (and thus using the S3 pointer rather than the full S3 content).
The lifecyle is actually: 
> idempotency (lock the idempotency key) => large message (get S3 content) => function code (`pjp.proceed`) => end of large message code (delete s3 file) => end of idempotency code (store the result in DB)

Regarding the s3 file, I'm actually not sure what can happen:
- If get from S3 or function code fails or deletion fails, the file is not deleted, so we can retry...
- If deletion succeed but end of idempotency fails, ... it's more an idempotency stuff than large message actually. If idempotency fails after the method succeeds, I don't remember what happens but it will be the same here

Reply:
> But whatever the order you choose for annotations on the method, Idempotency will always be the first executed (and thus using the S3 pointer rather than the full S3 content).

This is what I mean. I think we can stop at line 339:
> It means Idempotency module will use the initial raw message (containing the S3 pointer) and not the large message.

The rest, talking about what would happen if that _wasn't_ the case, is unnecessary detail for the user IMHO.